lib/Game.js

```diff
@@ -15,7 +15,7 @@ const PHASES = Object.freeze({
 class Game {
 	constructor({
 		pid = nanoid(),
-		players = ['Max', 'Tor'],
+		players = this.players || ['Max', 'Tor'],
```

Comment:
what should this line do? `this.players` should always be undefined here. `['Max', 'Tor']` already is the default for the parameter. If it is set, the value is overridden.

Reply:
Aha. I thought I needed it if the player didn't provide any `players` to the `.put`request. 
Will adjust it now